languages/german.php

```diff
@@ -116,3 +116,7 @@
 $lang["WITHPATH"] = "Mit Pfad:";
 $lang["FILEDELETED"] = "Datei gelöscht";
 $lang["FILEADDED"] = "Neue Datei";
+
+$lang['SEARCH'] = "suchen";
+$lang['SEARCH_PLACEHOLDER'] = "Suchen Sie hier nach Artikeln";
```

Comment:
This should be changed to `Suchen Sie hier nach Elementen`. `Artikel` doesn't make too much sense in this context for a native German, because it's more likely to be used as part of an online-shop or alike.

Reply:
Fixed it in the new commit.